lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -22,70 +22,37 @@
-    ('user__profile__cmeuserprofile__physician_status', 'Physician Status'),
-    ('user__profile__cmeuserprofile__patient_population', 'Patient Population'),
-    ('user__profile__cmeuserprofile__specialty', 'Specialty'),
-    ('user__profile__cmeuserprofile__sub_specialty', 'Sub Specialty'),
-    ('user__profile__cmeuserprofile__affiliation', 'Stanford Medicine Affiliation'),
-    ('user__profile__cmeuserprofile__sub_affiliation', 'Stanford Sub Affiliation'),
-    ('user__profile__cmeuserprofile__stanford_department', 'Stanford Department'),
-    ('user__profile__cmeuserprofile__sunet_id', 'SUNet ID'),
-    ('user__profile__cmeuserprofile__other_affiliation', 'Other Affiliation'),
-    ('user__profile__cmeuserprofile__job_title_position_untracked', 'Job Title or Position'),
-    ('user__profile__cmeuserprofile__address_1', 'Address 1'),
-    ('user__profile__cmeuserprofile__address_2', 'Address 2'),
-    ('user__profile__cmeuserprofile__city_cme', 'City'),
-    ('user__profile__cmeuserprofile__state', 'State'),
-    ('user__profile__cmeuserprofile__postal_code', 'Postal Code'),
-    ('user__profile__cmeuserprofile__county_province', 'County/Province'),
-    ('user__profile__cmeuserprofile__country_cme', 'Country'),
-    ('user__profile__cmeuserprofile__phone_number_untracked', 'Phone Number'),
-    ('user__profile__cmeuserprofile__gender', 'Gender'),
-    ('user__profile__cmeuserprofile__marketing_opt_in_untracked', 'Marketing Opt-In'),
-    ('user__id', '')
+    ('user__extrainfo__birth_date', 'Birth Date'),
+    ('user__extrainfo__professional_designation', 'Professional Designation'),
+    ('user__extrainfo__license_number', 'Professional License Number'),
+    ('user__extrainfo__license_country', 'Professional License Country'),
+    ('user__extrainfo__license_state', 'Professional License State'),
+    ('user__extrainfo__physician_status', 'Physician Status'),
+    ('user__extrainfo__patient_population', 'Patient Population'),
+    ('user__extrainfo__specialty', 'Specialty'),
+    ('user__extrainfo__sub_specialty', 'Sub Specialty'),
+    ('user__extrainfo__affiliation', 'Stanford Medicine Affiliation'),
+    ('user__extrainfo__sub_affiliation', 'Stanford Sub Affiliation'),
+    ('user__extrainfo__stanford_department', 'Stanford Department'),
+    ('user__extrainfo__sunet_id', 'SUNet ID'),
+    ('user__extrainfo__other_affiliation', 'Other Affiliation'),
+    ('user__extrainfo__job_title', 'Job Title or Position'),
+    ('user__extrainfo__address_1', 'Address 1'),
+    ('user__extrainfo__address_2', 'Address 2'),
+    ('user__extrainfo__city', 'City'),
+    ('user__extrainfo__state', 'State'),
+    ('user__extrainfo__postal_code', 'Postal Code'),
+    ('user__extrainfo__county_province', 'County/Province'),
+    ('user__extrainfo__country', 'Country'),
+    ('user__extrainfo__phone_number_untracked', 'Phone Number'),
+    ('user__profile__gender', 'Gender'),
+    ('user__extrainfo__marketing_opt_in_untracked', 'Marketing Opt-In'),
```

Comment:
Why are we renaming these? This still remains a fork with this commit applied, so wouldn't it be better to keep the values that state it's for CME?

Reply:
@caesar2164 No, that's not what's happening.

We used to default `PROFILE_FIELDS` to the columns for the old app,
then selectively override if the new app were installed.

Now we just default to the new app values.

Reply:
ah ok.